editoast/src/core/simulation.rs

```diff
@@ -66,25 +66,48 @@ pub struct PhysicsRollingStock {
     pub raise_pantograph_time: Option<u64>,
 }
 
-impl From<RollingStockModel> for PhysicsRollingStock {
-    fn from(value: RollingStockModel) -> Self {
+#[derive(Debug, Default)]
+pub struct SimulationParameters {
+    pub total_mass: Option<f64>,
+    pub total_length: Option<f64>,
+    pub max_speed: Option<f64>,
+}
+
+impl PhysicsRollingStock {
+    /// Create a new consist with simulation parameters
```

Comment:
Maybe we should remove that comment ? What the `new` method is doing is quite self-explanatory and the current comment is unclear.